pages/es/lb2/Command-line-tools.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Herramienta de Linea de Comandos"
+lang: es
+layout: translation
+keywords: LoopBack
+tags:
+sidebar: es_lb2_sidebar
+permalink: /doc/es/lb2/Command-line-tools.html
+summary: Éste artículo resume la herramienta CLI de LoopBack.
+translated: true
```

Comment:
i'm not sure if `translated` and `trans_complete` are needed. @nabdelgadir, could you please confirm? Thanks.

Reply:
No `translated` isn't needed here, only in the sidebar, and `trans_complete` should be used if it's not completely translated.